bundles/org.openhab.binding.electroluxair/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -51,6 +51,8 @@
 			<channel id="fanSpeed" typeId="fanSpeed"/>
 			<channel id="workMode" typeId="workMode"/>
 			<channel id="ionizer" typeId="ionizer"/>
+			<channel id="uiLight" typeId="uiLight"/>
+			<channel id="safetyLock" typeId="safetyLock"/>
```

Comment:
Can we please remove these new channels in the 3.4.x hotfix version? Only bugfixes should be merged, and this will also ensure that managed things will be upgraded correctly when upgrading to 4.0.

Reply:
Fixed!

Reply:
They seem to be still here, but now referring to deleted channel types. Perhaps a commit was not pushed?